core/shared/src/main/scala/org/http4s/multipart/Boundary.scala

```diff
@@ -41,9 +42,10 @@ object Boundary {
   private val rand = new Random()
 
   private def nextChar = CHARS(rand.nextInt(nchars - 1))
-  private def stream: LazyList[Char] = LazyList.continually(nextChar)
-  //Don't use filterNot it works for 2.11.4 and nothing else, it will hang.
-  private def endChar: Char = stream.filter(_ != ' ').headOption.getOrElse('X')
+  private def stream: LazyList[Char] =
+    LazyList.continually(nextChar)
+  // Don't use filterNot it works for 2.11.4 and nothing else, it will hang.
```

Comment:
Unrelated but I think this comment is out of date 😅

Reply:
Do you happen to know what the original issue was?

Reply:
No idea, I guess it was fixed in 2.11.4 though? We could try changing it to filterNot and see if the tests hang, but that doesn’t need to happen in this PR

Reply:
Ohh. I was reading it the other way, that _after_ 0.11.4 it's broken. Yeah I should but I was feeling lazy :P